rskj-core/src/main/java/co/rsk/config/ConfigLoader.java

```diff
@@ -128,4 +150,52 @@ private Config getNetworkDefaultConfig(Config userConfig) {
         logger.info("Network not set, using mainnet by default");
         return ConfigFactory.load(MAINNET_RESOURCE_PATH);
     }
+
+    private static void verify(String key, @Nullable ConfigValue expectedValue, ConfigValue actualValue) {
+        if (expectedValue == null) {
+            throw unexpectedKeyException(key, actualValue);
+        }
+
+        switch (actualValue.valueType()) {
+            case OBJECT:
+                if (!expectedValue.valueType().equals(ConfigValueType.OBJECT)) {
```

Comment:
I would suggest doing like : "Enum.VALUE.equals(value)", 
good practice

Reply:
well.. there're pros and cons.. people from sonarsource suggest using `==` - https://rules.sonarsource.com/java/RSPEC-4551.